admin_manual/installation/apps_management_installation.rst

```diff
@@ -47,32 +47,32 @@ If they are, then they will not be installed.
 Using Custom App Directories
 ----------------------------
 
-Use the **apps_paths** array in ``config.php`` to set any custom apps directory locations. 
-The key **path** defines the absolute file system path to the app folder. 
-The key **url** defines the HTTP web path to that folder, starting at the ownCloud web root. The key **writable** indicates if a user can install apps in that folder.
-
-.. note:: 
-   To ensure that the default **/apps/** folder only contains apps shipped with ownCloud, follow this example to setup an **/apps2/** folder which will be used to store all other apps.
-
-.. code-block:: php
-
-  <?php
-
-    "apps_paths" => array (
-        0 => array (
-                "path"     => OC::$SERVERROOT."/apps",
-                "url"      => "/apps",
-                "writable" => false,
-        ),
-        1 => array (
-                "path"     => OC::$SERVERROOT."/apps2",
-                "url"      => "/apps2",
-                "writable" => true,
-        ),
-    ),
+If you want to store apps in a custom directory, instead of ownCloud’s default (``/app``), you need to modify the ``apps_paths`` element in ``config/config.php``.
```

Comment:
I would add the reason why such an approach is a good idea, text as an example. 

1. You separate core apps from user/admin downloaded apps to ease distinguishing
2. Core apps are downloaded by core preferrable to the apps folder
3. Core may add new core related apps which are managed or downloaded during the installation process. Apps not used by core anymore may stay then as orphans in the apps folder
4. When you upgrade the instance and extract the new code into an empty folder, copying/aliasing the data folder, the config.php file ect (for details see upgrading...) you will also need to copy all manually downloaded apps. Having those in an separate directory eases the process of doing so because you then just need to take care on the folder.

Reply:
Well said.

Reply:
@mmattel, what do you mean by: _"Core apps are downloaded by core preferrable to the apps folder"_?